File: src/main/java/com/train/Ticket.java
package com.train;

public class Ticket {

    int ticketSum = 0;
    int rtSum = 0;
    public Ticket(){

    }
    public Ticket(int ticketSum,int rtSum){
        //V2使用
        this.ticketSum = ticketSum;
        this.rtSum = rtSum;
    }
    public int moneySumV1(int ticketSum,int rtSum){
        return (ticketSum-rtSum)*1000+rtSum*1800;
    }
    public String moneySumV2(){
        if (ticketSum<rtSum){
            return new String("Input error ticket numbers!!");
        }
        return Integer.toString(((ticketSum - rtSum) * 1000 + rtSum * 1800));
    }

}

File: src/main/kotlin/com/train/Ticket.kt
package com.train.kt

class Ticket(var ticketSum: Int=0,var rtSum: Int=0) {
    fun moneySumV1(ticketSum: Int, rtSum: Int): Int {
        return (ticketSum-rtSum)*1000+rtSum*1800
    }
    fun moneySumV2(): String {
        return if (ticketSum<rtSum) "Input error ticket number!!"
                else ((ticketSum-rtSum)*1000+rtSum*1800).toString()
    }
}